tools/sync.js

```diff
@@ -22,7 +22,10 @@ var listenBlocks = function(config) {
     var newBlocks = web3.eth.filter("latest");
     newBlocks.watch(function (error,latestBlock) {
     if(error) {
-        console.log('Error: ' + error);
+      console.log('Error: ' + error);
```

Comment:
I totally understand, it can get very frustrating when things aren't consistent like in other files. I'll do a clean up later to make all indents the same. For now, as shown in the following screenshot: 
<img width="223" alt="screen shot 2018-11-19 at 9 14 48 am" src="https://user-images.githubusercontent.com/9094204/48712399-ad244280-ebdb-11e8-8d2c-e595459079d7.png">
Original `console.log` looks like a 4-space indent while yours is a 2-space indent. In this case, I'd like it to be consistent, so 4-space indent.

Thank you so much for your help, I know it's very minor but we need to maintain consistency. I'll be cleaning up the code base moving forward.

Reply:
oh... I've missed before, in this case, `tools/sync.js` is 2-spaced indented and 4-space indent seems typo.

Reply:
Oh, you're right! The rest is 2-space indent. All good, no worries, resolving this convo :)